nerfstudio/data/datamanagers.py

```diff
@@ -240,12 +244,18 @@ class VanillaDataManagerConfig(InstantiateConfig):
     """Specifies the dataparser used to unpack the data."""
     train_num_rays_per_batch: int = 1024
     """Number of rays per batch to use per training iteration."""
-    train_num_images_to_sample_from: int = -1
+    train_num_images_to_sample_from: int = 300
```

Comment:
I change the default value here to handle large datasets better to avoid OOM.

If anyone does not like this, I can revert to the default value.